python/ray/tests/test_autoscaler.py

```diff
@@ -1708,6 +1772,85 @@ def do_nothing(*args, **kwargs):
         autoscaler.update()
         self.waitFor(lambda: len(runner.calls) > 0)
 
+    def testScaleDownMaxWorkers(self):
```

Comment:
Could use a short docstring and/or a couple comments in the code indicating the test's strategy.

Reply:
> Could use a short docstring and/or a couple comments in the code indicating the test's strategy.

done